pennylane_ls/MultiQuditDevice.py

```diff
@@ -41,14 +51,14 @@ def operations(self):
     def observables(self):
         return set(self._observable_map.keys())
 
-    def __init__(self, wires=1,shots=1, username = None, password = None):
+    def __init__(self, wires=1,shots=1, url = None, username = None, password = None):
         """
         The initial part.
         """
         super().__init__(wires=wires,shots=shots)
         self.username = username
         self.password = password
-        self.url_prefix = "http://qsimsim.synqs.org/multiqudit/"
+        self.url_prefix = url 
```

Comment:
I really dislike this. At least give a default value if the user is not choosing. They should not have to know each url to use pennylane no ?